t/18-backend-qemu.t

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/perl
+
+use 5.018;
+use strict;
+use warnings;
```

Comment:
why not `use Mojo::Base -base` ?

Reply:
I would not use that base class just for enabling strict and warnings.

Reply:
I thought it makes in general good default settings (feature, utf8 etc). Maybe not `Mojo::Base -base` but `Mojo::Base -strict`.
Just asking for best practice, as I currently thought it is preferred instead of using all the same `use` statements in each file.

Reply:
not sure, I just used the same as in t/18-qemu.t

Reply:
I would merge it as-is. If we decide to change this we can do it later and consistently for all tests.

Reply:
I see this as the necessary formal approval of this PR then :)